kindelia_core/src/events.rs

```diff
@@ -75,8 +76,9 @@ pub enum AddBlockEvent {
     siblings: Vec<Hash>,
     work: Hash,
   },
-  Computed {
-    blocks: Vec<Hash>,
+  ComputedBlock {
```

Comment:
Why rename this specific variant? `Block` seems redundant as it's a variant of `AddBlockEvent`.

Reply:
I only changed this because in one of our informal conversations you called this desired event `ComputedBlock`. But I will rename it back to only `Computed`